routers/api/v1/repo/repo.go

```diff
@@ -395,21 +395,28 @@ func Migrate(ctx *context.APIContext, form auth.MigrateRepoForm) {
 		return
 	}
 
+	var gitServiceType = models.PlainGitService
+	// TODO: this should be chosen by UI when from a customerize git service domain
+	if strings.HasPrefix(remoteAddr, "https://github.com") || strings.HasPrefix(remoteAddr, "http://github.com") {
```

Comment:
Same considerations as above. Perhaps there should be a `util` pkg function that does this? I've seen this `if()` all over the code.